### DISCUSSION TITLE
Keyboard Shortcut List and Another Capablities of Memos

### DISCUSSION THREAD

**Discussion by wronglylightcig** [usememos/memos#4185] [Ideas]
It could a list in Setting or accessible via a shortcut.

This also shows the capabilities of Memos itself, like oh I could do italic or bold. I have always had a good experience with the webapp, but I have no idea what's there or what possible, like: can I do super/sub-script? Or make a headline? If I can, how?

ps: if the feature of sub/super-scripting or headline is not already deployed, that would be a great addition!

**Comment by RoccoSmit** [comment]
You can find info on all the items you mentioned above in the docs [here](https://www.usememos.com/docs/getting-started/content-syntax)
